vispy/gloo/glir.py

```diff
@@ -233,7 +233,7 @@ def convert_shaders(convert, shaders):
                 lines.append(line.rstrip())
             # Write
             if not has_version:
-                lines.insert(0, '#version 120\n#line 2\n')
```

Comment:
It was in a block of lines that were yours so I bet it was part of a big copy-paste, don't worry about it